customer/test/test_customer_repository.py

```diff
@@ -13,6 +13,8 @@ def customer_repository():
 @pytest.mark.django_db
 class TestCustomerDjangoRepository:
     def test_create(self, customer_repository):
+        assert customer_models.Customer.objects.count() == 0
```

Comment:
Added as a sanity check to the test. I.e. we make sure there are no customers to begin with so we are sure the one found later in the test was added by the call. 

The only reason this would ever be necessary is because of something usually called "state bleed" or "state leak" between tests. Each test should start with a blank slate, so you can be sure that any change is a result of the code called in the test. If that breaks, it can lead to some very nasty issues e.g. tests passing and failing unpredictably because of the order in which they run.